pkg/mimir/modules.go

```diff
@@ -67,7 +67,7 @@ const (
 	ActivityTracker            string = "activity-tracker"
 	API                        string = "api"
 	SanityCheck                string = "sanity-check"
-	Ring                       string = "ring"
+	IngesterRing               string = "ingester-ring"
```

Comment:
Note to reviewers: I think this is not a breaking change because the module is configured as `modules.UserInvisibleModule`.